plaso/storage/sqlite/sqlite_file.py

```diff
@@ -431,6 +431,78 @@ def _ReadAndCheckStorageMetadata(self, check_readable_only=False):
     self.serialization_format = metadata_values['serialization_format']
     self.storage_type = metadata_values['storage_type']
 
+  def _UpdateEventDataIdentifierAfterDeserialize(self, event):
+    """Updates the event data identifier of an event after deserialization.
+
+    Args:
+      event (EventObject):
+    """
+    row_identifier = getattr(event, 'event_data_row_identifier', None)
+    if row_identifier is not None:
+      event_data_identifier = identifiers.SQLTableIdentifier(
+          'event_data', row_identifier)
+      event.SetEventDataIdentifier(event_data_identifier)
+
+      delattr(event, 'event_data_row_identifier')
+
+  def _UpdateEventDataIdentifierBeforeSerialize(self, event):
+    """Updates the event data identifier of an event before serialization.
+
+    Args:
+      event (EventObject):
+
+    Raises:
+      IOError: if the event data identifier type is not supported.
+      OSError: if the event data identifier type is not supported.
+    """
+    event_data_identifier = event.GetEventDataIdentifier()
+
+    # TODO: change to no longer allow event_data_identifier is None
```

Comment:
Is this todo still required? I thought all parsers were setting event data now?

Reply:
I'll address this in a later CL, wanted to keep it out of this one.